### ISSUE TITLE
nftables  vs nftables by netfilter

### ISSUE THREAD

**Issue by wangzhipeng** [google/nftables#67]
nftables  vs nftables by netfilter   diff ?

**Comment by stapelberg** [comment]
I don’t understand what you’re asking. Can you rephrase your question with more words please?

**Comment by wangzhipeng** [comment]
@stapelberg   What's the difference between this project and [nftables](https://netfilter.org/projects/nftables/index.html)

**Comment by stapelberg** [comment]
This project is a Go package to programmatically work with nftables.
